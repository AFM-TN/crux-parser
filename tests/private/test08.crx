// syntax error

let t[5] = x;
