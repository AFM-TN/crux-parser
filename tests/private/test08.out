Error parsing file.
SyntaxError(3,1)[Expected EOF but got LET.]
SyntaxError(3,1)[Could not complete parsing.]
